sdk/python/feast/feature_store.py

```diff
@@ -1013,6 +1020,23 @@ def _entity_row_to_field_values(
     return result
 
 
+def _use_time_feature_views(all_feature_views, features):
+    """Look for use-time FeatureView.join_key_maps and update all_feature_views"""
+    if isinstance(features, FeatureService):
+        feature_view_replacements = {
+            feature.name: feature
+            for feature in features.features
+            if isinstance(feature, FeatureView) and feature.join_key_map
```

Comment:
shoot, just realized these are all `FeatureViewProjection` objects cause it converts the features after initializing a FeatureService...

Reply:
This is dependent on a PR by @mavysavydav to have FeatureService.features allow FeatureView as one of the types.